lib/src/data/conversation.dart

```diff
@@ -24,6 +24,8 @@ class Conversation implements Comparable<Conversation> {
   DateTime lastUpdate;
   /// Status of the conversation
   bool isRead;
+    /// Whether the conversation is blocked
+  bool isBlocked;
```

Comment:
You should be able to safely delete this, because now the view model relies on `UserProfile.blockedConversations`